test/opcodes/bne_test.c

```diff
@@ -0,0 +1,36 @@
+#include "unity.h"
+#include "cpu.h"
+
+struct BUS bus;
+struct CPU cpu;
+
+void setUp(void) {
+  bus = initializeBus();
+  writeBus(0xFFFC, 0x00, &bus);
+  writeBus(0xFFFD, 0x06, &bus);
+  resetCpu(&cpu, &bus);
+}
+
+void tearDown(void) {
+}
+
+void should_change_program_counter_with_a_new_location_when_zero_flag_is_not_true(void) {
+  writeBus(0x600, 0xD0, &bus);
```

Comment:
D0 is BNE? And FF the location the cursor should jump if the flag zero is set?

Reply:
0xff = -1
So, when the flag zero = false and because BNE is relative `getAddressByOpcode` will add PC + 0xff.
Then:
0x601 + 0xff = 0x600

BNE will change PC to 0x600 then we have a jump to there.

Reply:
But yeah, `0xD0` is BNE opcode representation, I will change that into a var.